scripts/autofarm.deploy.ts

```diff
@@ -67,9 +79,16 @@ export async function deployAutofarmDiamond(args: any) {
     },
     abi: JSON.parse(String(diamond.interface.format('json'))),
   };
+
+  fs.mkdir(`./build/${CHAIN}/${ENV}/${name}`, { recursive: true }, (err) => {
+    if (err) console.error(err);
+  });
   fs.writeFileSync(
-    'Build/Diamond.json',
-    JSON.stringify(diamondFacetData, null, 2)
+    `./build/${CHAIN}/${ENV}/${name}/Diamond.json`,
+    JSON.stringify(diamondFacetData, null, 2),
+    (err) => {
+      if (err) console.error(err);
+    }
```

Comment:
- The same/similar function lines are being used repetitively in the code in many places.
Create a separate function and call that in each of those places.
- Also put the `fs.mkdir` and `fs.writeFileSync` in a `try-catch` block.
Reference - https://github.com/sumit03guha/Hardhat-BoilerPlate/blob/a2ea154fa5fee230bfc4904245087fc9648ded23/scripts/deploy.ts#L41